### DISCUSSION TITLE
hugo crash whith cached module

### DISCUSSION THREAD

**Discussion by kGk0** [hbstack/.github#27] [Issues and bugs]
Hello
First, thank you for this promising framework.
I used it several hours without trouble and then I got this error:

hugo: collected modules in 1014 ms
**ERROR render of "page" failed**: "_[...}_\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\docs@v0.6.4\layouts\docs\single.html:12:16": **execute of template failed**: template: docs/single.html:12:16: executing "main" at <partial "hb/modules/docs/doc/meta" .>: **error calling partial**: "_[...}_\AppData\Local\hugo_cache\modules\filecache\modules\pkg\mod\github.com\hbstack\docs@v0.6.4\layouts\partials\hb\modules\docs\doc\meta.html:3:26": **execute of template failed**: template: partials/hb/modules/docs/doc/meta.html:3:26: executing "partials/hb/modules/docs/doc/meta.html" at <site>: **invalid value; expected string**

I have cleared the cache and launched a new session but the result is the same. Have I broken something with a bad config somehow ?

**Comment by razonyang** [comment]
Could you please provide the public repo for reproducing?

It probably was caused by the wrong configuration. For example,

```yaml
// params.yaml
hb:
  docs:
```

The configuration will remove all default parameters of `hb.docs`, if you don't want to specify any params, then comment the whole set.

```yaml
// params.yaml
hb:
  # docs:
```

**Reply by kGk0** [reply]
I do not use any public repo, all what I do is local.
You spotted the trouble ! **docs:** was empty, I commented it out and ... tada, that works again !
Thanks a lot

**Reply by razonyang** [reply]
My pleasure.
